src/java/org/apache/cassandra/repair/RepairTask.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.cassandra.repair;
+
+import org.apache.cassandra.concurrent.ExecutorPlus;
+import org.apache.cassandra.utils.concurrent.Future;
+import org.apache.cassandra.utils.concurrent.ImmediateFuture;
+
+public interface RepairTask
+{
+    String getName();
```

Comment:
ATM there isn't a good way to monitor repair (this cleanup is to help with that work), so tools tend to need to use logging to figure out what's going on; so hesitant to change logging.

Reply:
Speaking of style... `getX` is something I try to avoid as far as possible, though there remain some aficianados in the codebase. It's simply unnecessarily more verbose than `x()`

Reply:
get vs no get, would be good to finally define this in our style guides.  In java get has meaning for data classes and MBeans, so for C* this limits the required locations to configurations and metrics; this example does not fall into either camp.

Rather than try to flesh this out in a per PR case, let me write up a dev@ to talk about these two topics and actually update our style guides to reflect a preference.

Reply:
punting including dev@ for now; made the change to `name()`